scan/main.py

```diff
@@ -52,8 +57,23 @@ def run(self):
             memory=True,  # Enable memory usage for enhanced task execution
         )
 
-        result = _run_crew(crew)
-        return result
+        results = _run_crew(crew)
+        logger.info(f"Raw results: {results}")
```

Comment:
The logger lazily evaluates the interpolation and this can result in unexpected results with f-strings.

```suggestion
        logger.info("Raw results: %s", results)
````